src/alias.c

```diff
@@ -89,39 +90,37 @@ int AliasTable_Del(AliasTable *table, const char *alias, IndexSpec *spec, int op
   return REDISMODULE_OK;
 }
 
-IndexSpec *AliasTable_Get(AliasTable *tbl, const char *alias) {
+StrongRef AliasTable_Get(AliasTable *tbl, const char *alias) {
+  StrongRef ret = {0};
```

Comment:
We can consider adding a reference as a 64-bit type a generic dictionary can hold. Otherwise, we must somehow store the value, and the StrongRef is just a pointer.
if the underlying pointer of a strong ref is null, it means the ref is not valid (in this case, we couldn't find the alias).